instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/config.go

```diff
@@ -0,0 +1,55 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelaws
+
+import (
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type config struct {
+	TracerProvider trace.TracerProvider
+	Propagators    propagation.TextMapPropagator
```

Comment:
This does not seem to be a used. Is it needed?

If not, please remove it and the related `WithPropagators` option.

Reply:
bump

Reply:
yeah, propagator is useless, need to be rmoved.